tests/integration_tests/functional/test_metrics.py

```diff
@@ -45,6 +47,8 @@ def test_flush_metrics(test_microvm_with_api):
         'uart',
         'signals',
     ]
+    if platform.machine() != "aarch64":
```

Comment:
I would not label the snapshotting metrics as being x86_64 specific in logger/metrics. The build is fine that way too.

Reply:
I thought about this for a while and decided in the end to leave them only on the platform on which snapshotting is available now (and remove the `x86_64` label when we will have support on `aarch64` too). I think it's confusing to print metrics (even with `0` values) for a feature that it's not actually supported.